makelib/variables.mk

```diff
@@ -24,7 +24,12 @@ QUICKRMTOOL := $(TOOLSDIR)/quickrm
 GO_TEST_PACKAGES ?= ./...
 GO_TEST_FUNC_ARGS ?=
 
-GO_ENV := $(strip GOPATH="$(GOPATH)" $(if $(strip $(GOROOT)),GOROOT=$(strip $(GOROOT))))
+GO_ENV := \
```

Comment:
Maybe wrap `$(GOARCH)`, `$(CC)` and `$(CXX)` in double quotes and wrap whole assignment in the `$(strip)` function, like in the following:
```
GO_ENV := $(strip \
	GOARCH="$(GOARCH)" \
	CGO_ENABLED=1 \
	CC="$(CC)" \
	CXX="$(CXX)" \
	GOPATH="$(GOPATH)" \
	$(if $(strip $(GOROOT)),GOROOT=$(strip $(GOROOT))))
```
(I hope I balanced the parens correctly here.)